detector/fcal/BeamCal_o1_v01_geo.cpp

```diff
@@ -45,6 +50,10 @@ static DD4hep::Geometry::Ref_t create_detector(DD4hep::Geometry::LCDD& lcdd,
   const double bcalInnerR = dimensions.inner_r();
   const double bcalOuterR = dimensions.outer_r();
   const double bcalInnerZ = dimensions.inner_z();
+  if (bcalInnerZ/dd4hep::mm > 3200) {
```

Comment:
Well, you need this warning for ILD.

Write a test that checks the position of BeamCal in the ILD models that throws an error if the BeamCal is too far in the back. Also should you then not check the end of BeamCal?